server/utils/utils.ts

```diff
@@ -36,6 +40,10 @@ export const compareByFullName = <T extends Person>(a: T, b: T): number => {
   return result !== 0 ? result : a.firstName.localeCompare(b.firstName)
 }
 
+export const compareByDateAndTime = <T extends DateAndTime>(a: T, b: T): number => {
+  return moment(a.movementDateTime).unix() - moment(b.movementDateTime).unix()
+}
```

Comment:
this is a little bit of the domain creeping into otherwise domain agnostic code.

could possibly handle this by saying you have something of type `T` 
and a mechanism (function) of getting a date from it: `T => Date`

Reply:
sorted in latest PR